tests/test.py

```diff
@@ -63,5 +67,24 @@ def test_predict():
         r = requests.post(url=model_endpoint, files=file_form)
     assert r.status_code == 400
 
+def test_img_resize():
+        """
+        The image resize test.
+        """
+
+        resize_path=[]
+        for file in os.listdir("."):
```

Comment:
This test is not actually doing anything as `resize_path` will be an empty list `[]`. So none of the assertions in the `for` loop will be tested.

```suggestion
        for file in os.listdir("tests/"):
```

Reply:
Thanks @MLnick.